app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -371,9 +371,11 @@ angular
           .then(
             (pipelines) => {
               const latestFromServer = pipelines.find((p) => p.id === toSave.id);
-              const latest = latestFromServer || toSave;
-              setOriginal(latest);
-              this.revertPipelineChanges();
+              if (latestFromServer && latestFromServer.updateTs) {
+                toSave.updateTs = latestFromServer.updateTs;
```

Comment:
simplify the updateTs checking

Apparently there is some nasty angular/react glue that is causing updateTs to get clobbered when the `revertPipelineChanges` method was being used.